src/components/footer.tsx

```diff
@@ -1,15 +1,19 @@
 import Link from "next/link";
 import Logo from "./logo";
 import TextReveal from "./ui/text-reveal";
-import { DiscordLogoIcon, GitHubLogoIcon, TwitterLogoIcon } from "@radix-ui/react-icons";
+import {
+	DiscordLogoIcon,
+	GitHubLogoIcon,
+	TwitterLogoIcon,
+} from "@radix-ui/react-icons";
 import { MastodonLogo } from "./icons/mastodon";
 import { Button } from "./ui/button";
 
 export default function Footer() {
 	return (
 		<div className="border-grey align-center mt-10 flex w-full flex-col border-t px-10 py-10 font-medium md:px-0">
 			<div className="mx-auto flex w-full justify-between border-b px-10 pb-10 pt-10 lg:!w-2/3 lg:px-0">
```

Comment:
I would suggest adding gap here instead of margin to its child.

Reply:
gap-[20px] to the parent block
![image](https://github.com/user-attachments/assets/af2d6da2-3789-4bc6-a3f5-a23958f5e771)

Reply:
Changed it. Thanks for the suggestion!